### ISSUE TITLE
Suikoden 3 - Top half of screen has garbo

### ISSUE THREAD

**Issue by RedDevilus** [PCSX2/pcsx2#3849]
Source: https://forums.pcsx2.net/Thread-Bug-Report-1-6-0-Suikoden-3

![image](https://user-images.githubusercontent.com/24227051/96374938-233dd500-1176-11eb-851d-d0591e39e2f7.png)

Needs research for the reason of this problem.

Savestates with 1.7 build: 
https://mega.nz/file/FoZ0AJCC#PxH7OP0CZrqqfq38bEbmq5hBhfMxVqTcw3gX9Tan7cU

**Comment by tadanokojin** [comment]
Still needs research from me but basic idea is there is no TS, instead post-process is done on the EE and done it two halves. a 16 bit depth buffer is repurposed as a scratch buffer to upload to the EE (this is the last thing done, so they don't care if they destroy it).

Bottom halve is classic cache issue, game increments the base pointer to do the bottom and the cache can't deal with it.

**Comment by JordanTheToaster** [comment]
Still an issue in 1.7.2724 even in software it seems to break.

![image](https://user-images.githubusercontent.com/80843560/167619298-390e748e-6c5f-4a79-843d-d27402fdc4dd.png)

**Comment by Mrlinkwii** [comment]
hows this on latest nightly ?

**Comment by refractionpcsx2** [comment]
user confirmed on the forum that it's still bugged. https://forums.pcsx2.net/Thread-Bug-Report-1-6-0-Suikoden-3?pid=638419#pid638419

There is weird stuff going on in this game though.

**Comment by CatGreen90** [comment]
I tested it a few weeks ago and it wasn't quite as bad as before (no stripes, just grayscale on the top half and color on the bottom).

![Suikoden III_SLUS-20387_20231201213511](https://github.com/PCSX2/pcsx2/assets/44750311/b112718c-4314-4a12-af71-643d056176be)

**Comment by refractionpcsx2** [comment]
yeah, it's doing the shuffle in two halves, so the top half works, then the bottom half misses and doesn't happen.

at least in this scenario

**Comment by bigol83** [comment]
> yeah, it's doing the shuffle in two halves, so the top half works, then the bottom half misses and doesn't happen.
> 
> at least in this scenario
> 
> We could use some GS dumps if people want this fixed :P

There are several gsdumps with the latest build here https://forums.pcsx2.net/Thread-Bug-Report-1-6-0-Suikoden-3?pid=638419#pid638419

last post

**Comment by refractionpcsx2** [comment]
ah thanks, I hadn't rechecked the forum post :)

**Comment by bigol83** [comment]
Flashback scene is supposed to be black and white, i checked a Ps2 playthrough. Software now shows flashbacks correctly, hardware renderers have color that isn't supposed to be there. That graphic corruption happening in software too is alot less now, but it's still there.

**Comment by Mrlinkwii** [comment]
still an issue , its much better than it was , using v2.1.182
![image](https://github.com/user-attachments/assets/7bc665f2-bbc2-4ab6-973b-9fbe7faeeb42)

[issue_3849.zip](https://github.com/user-attachments/files/17250371/issue_3849.zip)

im gonna post the gs dump provided on the forum , in case it gets lost
